requirements.txt

```diff
@@ -1,11 +1,6 @@
-aniso8601==8.0.0
-click==7.1.2
 Flask==1.1.2
 Flask-Cors==3.0.8
 Flask-RESTful==0.3.8
-itsdangerous==1.1.0
-Jinja2==2.11.2
-MarkupSafe==1.1.1
+aniso8601==8.0.0
```

Comment:
requirements.txt is supposed to hold versions for implicit dependencies too. I can see how that helps avoid random bugs from upgrading a library without noticing.

I would be happy to use dependencies without versions too. That would be a useful way for development if we have over 95% test coverage. The high test coverage would then allow us catch any surprise bugs from library upgrades.